### PR TITLE
Changed cat to message

### PR DESCRIPTION
**Pull request by ldecicco-USGS**
To easily suppress the set_endpoint output (ie...reducing output text for Condor runs). Also minimally started reducing notes/warnings.

### CODE REVIEW

File: R/set_endpoint.R [hunk 1]
```diff
@@ -28,14 +28,14 @@ set_endpoint = function(endpoint="production"){
...
-		cat('Setting endpoint to www.sciencebase.gov\n');
+		message('Setting endpoint to www.sciencebase.gov');
 		pkg.env$url_base = "https://www.sciencebase.gov/catalog/"
 		
 	}else if(endpoint=="development" || endpoint=="dev"){
-		cat('Setting endpoint to beta.sciencebase.gov\n');
+		message('Setting endpoint to beta.sciencebase.gov');
 		pkg.env$url_base = "https://beta.sciencebase.gov/catalog/"
 	}else{
-		error('Unsupported endpoint option')
```
Comment (jread-usgs) [inline R/set_endpoint.R]: haha. whoops

**Comment by lawinslow** [comment]
I like it.

**Comment by jread-usgs** [comment]
fine by me. @lawinslow ?

**Comment by lawinslow** [comment]
Just waiting for judgementbot

**Pull request merged**